### PR TITLE
[Fix #54] Allow user to search for donor by NRIC/UEN

### LINKED ISSUE #54: Make NRIC searchable in `DonorsController#index`
Dawn needs to be able to search by NRIC in addition to name on the donors index.

The change itself is trivial, but we should make sure that we have an index on the NRIC column in the database table.

### PR DESCRIPTION
**Pull request by limxinyuuu**
This enhancement makes NRIC/UEN searchable in DonorsController#index.
---

**Before submitting, check that:**

 - [x] You have added (passing) tests for your code.
 - [x] You have written good* commit messages.
 - [x] You have squashed relevant commits together.
 - [x] You have ensured that RuboCop is passing.
 - [x] Your PR relates to one subject, with a clear title and
       description using complete, grammatically correct sentences.
---

*If you do not know what makes a good commit message, or why it
is important, please refer to these resources: [1][1], [2][2], [3][3].

[1]: https://robots.thoughtbot.com/5-useful-tips-for-a-better-commit-message
[2]: http://chris.beams.io/posts/git-commit/
[3]: https://github.com/blog/1943-how-to-write-the-perfect-pull-request

### CODE REVIEW

File: app/models/donor.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@ class Donor < ActiveRecord::Base
 
   validates :identification, presence: true, uniqueness: true
 
-  scope :search, -> (keyword) { where("name ILIKE ?", "%#{keyword}%") }
+  scope :search, -> (keyword) { where("name ILIKE ? OR  identification ILIKE ?", "%#{keyword}%", "%#{keyword}%") }
```
Comment (Drenmi) [inline app/models/donor.rb]: You can use a "placeholder condition" here, so you don't have to repeat the keyword argument:

```
where("name ILIKE :keyword OR identification ILIKE :keyword", { keyword: "%#{keyword}%" })
```

**Comment by Drenmi** [comment]
LGTM! Want to try adding a test for searching by NRIC? 😀

**Comment by limxinyuuu** [comment]
@Drenmi I have added a test 😄

**Comment by Drenmi** [comment]
Wow! That is so awesome. 😀 

Ge ahead and squash your commits, and merge. 🌻 🌻 🌻

**Pull request merged**